src/components/MainApp/MainApp.tsx

```diff
@@ -44,6 +45,8 @@ export function MainApp(props: any) {
                 localStorage.setItem("lastReloadTime", new Date().getTime().toString());
             }
         }
+
+        startMigrations();
```

Comment:
I think this should not trigger automatically or at least tell the user in a way that he should double check the new blacklist